vocab/de-de/AdultMode.voc

```diff
@@ -0,0 +1 @@
+hurt me plenty
```

Comment:
Should be translated?

Reply:
i think not, this skill is basically english only, you can't translate the audio files only

being a easter eggs skill, this would also be cultural and very language dependent, if doom was translated to other langs then the easter egg holds, if not then the translation is meaningless and does not have the fun factor

I'd say these intents should not be translated at all, a non english query would have non english easter eggs